azext_iot/central/_help.py

```diff
@@ -396,6 +397,28 @@ def _load_central_device_twin_help():
     """
 
 
+def _load_central_dps_keygen_help():
+    helps[
+        "iot central dps-keygen"
```

Comment:
actually on second thought - we should actually make the entry point for this be in `dps` shouldn't we? cc: @digimaun 

Something like

`az iot dps compute-derived-sas-key`

Reply:
I also think this further adds to the argument that this shuold be placed in utility or somewhere more common for consumption

Reply:
If this is a general solution then I agree it shouldn't be in the iot central command space. I could see it landing in `az iot dps` or `az iot device`. Need to brainstorm more on the actual command name, not crazy about `compute-derived-sas-key`.